components/automatic-points-table/commons/automatic-points-table.lua

```diff
@@ -67,18 +69,43 @@ function AutomaticPointsTable.run(frame)
 	return divTable:create()
 end
 
+function AutomaticPointsTable:storeLPDB(pointsData)
+	Table.iter.forEachIndexed(pointsData, function(index, teamPointsData)
+		local team = teamPointsData.team
+		local teamName = string.lower(team.aliases[#team.aliases])
+		local lpdbName = self.parsedInput.lpdbName
+		local uniqueId = teamName .. '_' .. lpdbName
+		local position = teamPointsData.position
+		local totalPoints = teamPointsData.totalPoints
+		local objectData = {
+			type = lpdbName,
+			name = teamName,
+			information = position,
+			date = os.date(),

```

Comment:
This will give (possibly) give you a slightly different date for each iteration. Do this before the loop

Reply:
```suggestion
			date = date,
```